docs/articles/documentation/test-api/test-code-structure.md

```diff
@@ -64,6 +65,44 @@ If the start page is not specified, it defaults to `about:blank`.
 > You can also call the [Navigate action](actions/navigate.md) at the beginning of a test
 > to navigate to the start page.
 
+### Http Authentication
+
+TestCafe allows you to test web pages that are protected with Http Basic or Windows (NTLM) authentication. To set authentication credentials use the `httpAuth` fixture declaration method.
+
+```text
+httpAuth( credentials )
+```
+
+Parameter     | Type   | Description
+------------- | ------ | ------------------------------------------------
+`credentials` | Object | Object that includes all necessary data for the authentication.
+
+Credentials parameter has following format:
+
+Parameter     | Type   | Description
+------------- | ------ | --------------------------------------------------------------------
+`username`    | String | Username for the authentication.
+`password`    | String | Password for the authentication.
+`workstation` | String | PC's ID in the local network (optional parameter).
+`domain`      | String | Domain name (optional parameter).
```

Comment:
in other places we describe optional parameters in the following way:

`workstation` _(optional)_ | String | PC's ID in the local network.
`domain` _(optional)_     | String | Domain name.